package/gluon-offline-ssid/luasrc/usr/bin/gluon-offline-ssid

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/lua
+
+local unistd = require 'posix.unistd'
+local uci = require("simple-uci").cursor()
+local util = require 'gluon.util'
+local site = require 'gluon.site'
+
+-- -------------
+-- safety checks
+-- -------------
+
+-- TODO:
+-- pgrep -f autoupdater >/dev/null && safety_exit 'autoupdater running'
+
+function safety_exit(t)
+  io.write(t .. ", exiting with error code 2")
+  os.exit(2)
+end
+
+function shell_exec(command)
+  local file = assert(io.popen(command, 'r'))
+  local output = file:read('*all')
+  file:close()
+  return output
+end
+
+function file_exists(name)
+   local f=io.open(name,"r")
```

Comment:
Use spaces before and after equal sign and after a comma

Reply:
I'm not sure, but I think you can write return assert(io.popen(command, 'r')):read('*a') instead in one line.

Reply:
rewritten